source/tyr/migrations/versions/465a7431358a_add_import_ntfs_to_mimir.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: 465a7431358a
+Revises: 3acdde6329db
+Create Date: 2018-02-23 14:50:23.086805
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '465a7431358a'
+down_revision = '3acdde6329db'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('instance', sa.Column('import_ntfs_in_mimir', sa.Boolean(), nullable=True))
```

Comment:
you can try this:

`op.add_column('instance', sa.Column('import_ntfs_in_mimir', sa.Boolean(), nullable=False, server_value='false'))`